packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -150,12 +150,24 @@
   {{! template-lint-disable require-input-label }}
   <Shw::Grid @columns={{4}} as |SG|>
     <SG.Item @label="currentLength = 0">
-      <input type="text" aria-label="currentLength = 0" id="cc-cl-eq-0" />
-      <Hds::Form::CharacterCount @controlId="cc-cl-eq-0" />
+      <input
+        type="text"
+        aria-label="currentLength = 0"
+        id="cc-cl-eq-0"
+        value={{this.value1}}
+        {{on "input" (fn this.updateValue "value1")}}
+      />
+      <Hds::Form::CharacterCount @value={{this.value1}} @controlId="cc-cl-eq-0" />
```

Comment:
Is the `@controlId` still needed? 🤔

Reply:
If not (as I suspect) than also the function that generates the `id()` can be updated to generate a generic ID (like we do for other similar field sub-elements)

Reply:
This and the `onInsert` are still needed to hook up the `aria-describedby` properly - I initially removed both of them but tests caught the breakage and I backed out the removals.